### ISSUE TITLE
Kirki styles and scripts aren't properly enqueue

### ISSUE THREAD

**Issue by Zjadlbymcos** [themeum/kirki#2303]
### Issue description:
Kirki generates bad URI for enqueue styles / script when theme files are not in theme root directory.

### Version used:
(Did you try using the develop branch from GitHub? There's a chance your issue has already been addressed there)

3.1.0 installed through wordpress (plugin)

### Using theme_mods or options?
No

### Code to reproduce the issue (config + field(s))
There is no code needed to reproduce the issue but I created repo with example code that help to reproduce issue: https://github.com/Zjadlbymcos/kirki-enqueue-issue

TLDR: The issue is when theme files (index.php, style.css and function.php) aren't in theme root dir.

for **themes/theme-name/style.css**  Kirki properly loads own styles: `/wp-content/plugins/kirki/modules/icons/icons.js?ver=3.1.0`

for **themes/theme-name/resources/style.css** - Kirki generated weird uri for styles:
`/wp-content/themes/plugins/kirki/modules/icons/icons.js?ver=3.1.0`

**Comment by Zjadlbymcos** [comment]
I found workaround to force Kirki to properly load styles/scripts. I created a plugin with with this code:

```php
add_filter('kirki_config', function(array $config) {
    $config['url_path'] = plugins_url('kirki');
    return $config;
}, 1, 1);
```

**Comment by mapsteps** [comment]
Hello and thank you for reporting this issue.

We are closing most open GitHub issues with the release of Kirki 4 to start fresh with this major release.

If you are still running into issues after you have tested your theme with Kirki 4, please let us know so we can reopen the issue.

Please note that Kirki 3 is no longer actively supported. Moving forward, we will only provide support the very latest version of Kirki.

Thank you for your understanding.

Best,
David
